test/test_tree_2d_kpp.jl

```diff
@@ -0,0 +1,21 @@
+module TestExamplesKPP
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+EXAMPLES_DIR = joinpath(examples_dir(), "tree_2d_dgsem")
+
+@testset "KPP" begin
+  @trixi_testset "elixir_kpp.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_kpp.jl"),
+      l2   = [5.17986197e-01],
+      linf = [8.93080899e+00],
+      tspan = (0.0, 0.1),
+      atol = 1e-6,
+      rtol = 1e-6)
```

Comment:
Are these tolerances really necessary (or asked differently, why aren't the default tolerances sufficient)?

Reply:
OK,. fair enough!